### PR TITLE
python3Packages.pyu2f: 0.1.4 -> 0.1.5a

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

- version update

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120904` at 0404e506 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyu2f</li>
    <li>python39Packages.pyu2f</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/development/python-modules/pyu2f/default.nix:1:3:
```
  |
1 | { stdenv, lib, fetchFromGitHub, buildPythonPackage,
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyu2f/default.nix [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
 
-  # Platform detection for linux fails
-  postPatch = lib.optionalString stdenv.isLinux ''
-    rm pyu2f/tests/hid/macos_test.py
-  '';
-
   propagatedBuildInputs = [ six ];
 
-  checkInputs = [ pytest mock pyfakefs unittest2 ];
+  checkInputs = [ mock pyfakefs pytest unittest2 ];
```
Comment (drewrisinger) [inline pkgs/development/python-modules/pyu2f/default.nix:24]: Could this be reduced to ``checkInputs = [ pytestCheckHook pyfakefs mock ];``? Could also remove the custom ``checkPhase``.

Unittest2 is only needed for python < 2.7, and mock is only needed for python < 3.3.

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [x] Diff LGTM, minus potential switch to ``pytestCheckHook``.
* [x] Commit LGTM
* [x] Builds via ``nix-review`` on x86-64-linux

```
2 packages built:
python38Packages.pyu2f python39Packages.pyu2f
```

File: pkgs/development/python-modules/pyu2f/default.nix (hunk 1 above)
Comment (prusnak) [reply pkgs/development/python-modules/pyu2f/default.nix]: Addressed in 16c8941c1325a7d61f0ca2258b0b484f854055bb

Comment (drewrisinger) [reply pkgs/development/python-modules/pyu2f/default.nix]: If you're effectively restricting build to python3 (as you seem to be implicitly doing), then it'd be worth removing ``mock`` as well. Might also be worth an explicit ``disabled = pythonOlder "3.3"``?

Comment (prusnak) [reply pkgs/development/python-modules/pyu2f/default.nix]: I tried building without mock and it failed.

Comment (drewrisinger) [reply pkgs/development/python-modules/pyu2f/default.nix]: Ah, it's probably an imports issue. e.g. the difference between (the equivalent) ``import unittest.mock`` & ``import mock``.

**Review by drewrisinger — APPROVED** [review]
* [x] Diff LGTM
* [x] Commits LGTM
* [x] Builds via ``nix-review`` on x86-64-linux

```
2 packages built:
python38Packages.pyu2f python39Packages.pyu2f
```

**Pull request merged**